circuit_maintenance_parser/parsers/lumen.py

```diff
@@ -39,7 +39,10 @@ def parse_spans(self, spans: ResultSet, data: Dict):
                 ):
                     data["maintenance_id"] = line_text.split("#: ")[-1]
                 elif line_text.startswith("summary:"):
-                    for sibling in line.next_siblings:
+                    siblings = line.next_siblings
+                    if not line.next_sibling:
```

Comment:
```suggestion
                    if not siblings:
```

Reply:
Had that initially, but `siblings` is actually a generator. So even if it's empty it's not `None`. Cleanest solution was to use the bs4 lib, which returns `None` when no sibling.

Reply:
sure, now I remember